go/teams/service_helper.go

```diff
@@ -453,3 +453,15 @@ func apiArg(ctx context.Context, endpoint string) libkb.APIArg {
 	arg.SessionType = libkb.APISessionTypeREQUIRED
 	return arg
 }
+
+func GetRootID(ctx context.Context, g *libkb.GlobalContext, id keybase1.TeamID) (keybase1.TeamID, error) {
+	team, err := g.GetTeamLoader().Load(ctx, keybase1.LoadTeamArg{
+		ID: id,
```

Comment:
Pass StaleOK to always accept the cache. Because the property you're using is immutable. :)